docker/Dockerfile

```diff
@@ -13,7 +13,8 @@ ARG PYTHON_VERSION=3.10
 ARG TORCH_VERSION=2.3.0
 ARG TORCHVISION_VERSION=0.18.0
 
-RUN rm /etc/apt/sources.list.d/cuda*.list && apt-get update -y && apt-get install -y software-properties-common wget vim &&\
+RUN apt-get update -y && apt-get install -y software-properties-common wget vim git curl &&\
+    curl https://sh.rustup.rs -sSf | sh -s -- -y &&\
```

Comment:
what's the motivation of `curl https://sh.rustup.rs`

Reply:
> what's the motivation of `curl https://sh.rustup.rs`

for install rust complier;when gradio>4.40.0,it will depend on  orjson that need rust compiler when installing; more and more pip package implementate by rust as time goes on